### PR TITLE
Update to 0.0.15

### PR DESCRIPTION
**Pull request by mak448a**

### CODE REVIEW

**Comment by mak448a** [comment]
YOu need to add darkdetect as a whl

**Comment by mak448a** [comment]
Sorry to keep bothering you, but do you know what could be causing this issue? @bbhtt

File: python3-modules.json [hunk 1]
```diff
@@ -108,6 +108,49 @@
...
+        {
+            "name": "darkdetect",
+            "buildsystem": "simple",
+            "build-commands": [
+                "pip3 install --verbose --exists-action=i --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} \"darkdetect\" --no-build-isolation"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "url": "https://files.pythonhosted.org/packages/f2/f2/728f041460f1b9739b85ee23b45fa5a505962ea11fd85bdbe2a02b021373/darkdetect-0.8.0-py3-none-any.whl",
```
Comment (bbhtt) [inline python3-modules.json:136]: This version should be **less** than 0.8.0, >=0.7.1

> `Could not find a version that satisfies the requirement darkdetect<0.8.0,>=0.7.1 `

Comment (mak448a) [reply python3-modules.json:136]: Thanks!

**Pull request merged**